.github/workflows/on_pr_opened.yaml

```diff
@@ -13,13 +19,12 @@ on:
       - synchronize
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  group: ${{ github.workflow }}-${{ github.event.inputs.pr-number || github.ref }}
```

Comment:
This will break any actions started with pull_request. I think we probably need a whole new action to do a workflow_dispatch to run all the things that need secrets.

Reply:
do you mean combine all of the workflows into a single action?